index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <mete name="viewport" contact="with=device-width, initial-scale=1.0">
+        <title>PATHOU'S calculator</title>
+        <link rel="stylesheet" href="style.css"/>
+        <script src="index.js"></script>
+
+    </head>
+    <body>
+        <form>
+            1st Number : <input type="text" id="firstNumber" /><br>
+            2nd Number : <input type="text" id="secondNumber" /><br><br>
+            <input type="button" onClick="addition()" Value="Addition" />
```

Comment:
The idea here was to make a drop down box for your operations.  It is cleaner more efficient code.